### PR TITLE
Add compat data for Clients

### PR DESCRIPTION
**Pull request by bunnybooboo**
https://developer.mozilla.org/en-US/docs/Web/API/Clients

as per #1002 note on ESR may need editing

### CODE REVIEW

File: api/Clients.json [hunk 1]
```diff
@@ -0,0 +1,302 @@
...
+              "version_added": "45"
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": null
+            },
+            "opera_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/Clients.json]: Both versions of Opera should be 38.

File: api/Clients.json [hunk 2]
```diff
@@ -0,0 +1,302 @@
...
+            },
+            "firefox": [
+              {
+                "version_added": "44",
+                "notes": "Service workers (and Push) have been disabled in the Firefox 45 & 52 Extended Support Releases (ESR)."
+              },
+              {
+                "version_added": "45",
+                "notes": "<code>includeUncontrolled</code> support."
```
Comment (Elchi3) [inline api/Clients.json]: The `includeUnControlled` option should be a sub feature of `matchAll`. (much like you added a subfeature `async_waitUntil ` to `waitUntil` in https://github.com/mdn/browser-compat-data/pull/1006#discussion_r167990620).

```
"includeUncontrolled_option": {
  "__compat": {
  "description": "<code>includeUncontrolled</code> option",
```
This way, a separate row will appear for this in the new table.

Thank you!

File: api/Clients.json [hunk 3]
```diff
@@ -236,6 +228,56 @@
...
+          "description": "<code>includeUncontrolled</code> option",
+          "support": {
+            "webview_android": {
+              "version_added": false
+            },
+            "chrome": {
+              "version_added": null
+            },
+            "chrome_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/Clients.json]: `matchAll()` landed in Chrome 47/Opera 32 and it included the `includeUncontrolled` option. Can you please fix throughout?

Comment (bunnybooboo) [reply api/Clients.json]: Why not, hey!

Comment (jpmedley) [reply api/Clients.json]: I apologize for not catching this yesterday. Thank you for doing this. I'm drinking from a fire hose when it comes to reviewing compat PRs. I really appreciate the help.

Comment (bunnybooboo) [reply api/Clients.json]: BTW are most cases of ServiceWorker APIs consistent across webview, chrome, and android? I'm not so clued on the release model.

Comment (jpmedley) [reply api/Clients.json]: There are minor differences here and there. I plan to run all of the sw data within the coming week.

Comment (jpmedley) [reply api/Clients.json]: In general, push notifications aren't supported in WebView. 99% of the time Webview, Android, and Chrome have the same version number.

File: api/Clients.json [hunk 4]
```diff
@@ -0,0 +1,350 @@
...
+                "notes": "Can only open URLs on the same origin."
+              },
+              {
+                "version_added": "43",
+                "notes": "Can open any URL. "
+              },
+              {
+                "version_added": "51",
+                "notes": " URLs may open inside an existing browsing context provided by a standalone web app"
+              }
```
Comment (jpmedley) [inline api/Clients.json]: All three versions of Chrome should look like this one.

File: api/Clients.json [hunk 5]
```diff
@@ -0,0 +1,357 @@
...
+            "deprecated": false
+          }
+        }
+      },
+      "openWindow": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Clients/openWindow",
+          "support": {
+            "webview_android": {
+              "version_added": {
```
Comment (Elchi3) [inline api/Clients.json]: This is what makes the build fail here:
```
 "version_added": {
   "version_added": "51",
```

Please remove `"version_added": {`

File: api/Clients.json [hunk 6]
```diff
@@ -0,0 +1,357 @@
...
+            }
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "includeUncontrolled_option": {
```
Comment (Elchi3) [inline api/Clients.json]: Thanks for adding this, however it needs to be nested underneath "matchAll".

- matchAll
  - __compat
  - includeUncontrolled_option

**Pull request merged**